src/olympia/devhub/tests/test_views.py

```diff
@@ -1380,7 +1380,8 @@ def test_experiment_xpi_not_allowed(self, mock_validator):
 
     @mock.patch('olympia.devhub.tasks.run_addons_linter')
     def test_system_addon_allowed(self, mock_validator):
-        user_factory(email='redpanda@mozilla.com')
+        user = user_factory(email='redpanda@mozilla.com')
```

Comment:
Would there be any value in using an email address that doesn't end with `mozilla.com`, just to prove that that is not what is allowing this test to pass?

Reply:
Yeah - I removed the email pretty much everywhere else to prove just that, but for some reason here I changed my strategy and kept the email but re-used it also in the test below where it's *not* allowed (because it doesn't have the permission). I guess being consistent and never having mozilla.com emails is best, I'll change it.

Reply:
8c8f9a8a9f6980c7dc702160d08d47e2c32ee42e